app/src/interfaces/group-detail/group-detail.vue

```diff
@@ -45,128 +45,82 @@
 	</v-detail>
 </template>
 
-<script lang="ts">
-import { Field } from '@directus/types';
-import { computed, defineComponent, PropType, ref, watch } from 'vue';
-import { ValidationError } from '@directus/types';
-import { useI18n } from 'vue-i18n';
+<script setup lang="ts">
 import formatTitle from '@directus/format-title';
+import { Field, ValidationError } from '@directus/types';
 import { isEqual } from 'lodash';
+import { computed, ref, watch } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+const props = withDefaults(
+	defineProps<{
+		field: Field;
+		fields: Field[];
+		primaryKey: number | string;
+		values: Record<string, unknown>;
+		initialValues: Record<string, unknown>;
+		disabled?: boolean;
+		batchMode?: boolean;
+		batchActiveFields?: string[];
+		loading?: boolean;
+		validationErrors?: ValidationError[];
+		badge?: string;
+		start?: 'open' | 'closed';
+		headerIcon?: string;
+		headerColor?: string;
+		direction?: string;
+	}>(),
+	{
+		batchActiveFields: () => [],
+		validationErrors: () => [],
+		start: 'open',
+	}
+);
+
+defineEmits(['apply']);
```

Comment:
Until now, we had every `defineEmits` typed. Let's continue to do so!

In this case it should be:

```ts
defineEmits<{
	(e: 'apply', value: Record<string, unknown>): void;
}>();
```